### PR TITLE
feature preview of image-objects

### PR DESCRIPTION
**Pull request by tweigel-dev**
## Description
This is a feature to show a preview of a Object if it is an image. Therefor i added a new objectAction for previewing the image. This actions opens a simple preview modal like share and delete. To check whether the object is an image i add a function at mime.js to check the content-type.

## Motivation and Context
When saving as objects in MinIO it could be very helpful to preview the image in the browser.

## How to test this PR?
I extended the test-cases to test the feature automatically.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation needed
- [ ] Unit tests needed
- [ ] Functional tests needed (If yes, add [mint](https://github.com/minio/mint) PR # here: )

### CODE REVIEW

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#941e33b3-af99-4f75-a0d0-651667145837) |
<a name="941e33b3-af99-4f75-a0d0-651667145837"></a>
### 9239-9d5364a/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c3.minio.io:30686
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 1d14468975f5:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 8 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 8 minutes and 55 seconds
(4/15) Running aws-sdk-ruby tests ... done in 18 seconds
(5/15) Running awscli tests ... done in 3 minutes and 1 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 6 minutes and 28 seconds
(8/15) Running minio-dotnet tests ... done in 1 minutes and 41 seconds
(9/15) Running minio-go tests ... done in 20 minutes and 42 seconds
(10/15) Running minio-java tests ... FAILED in 8 minutes and 28 seconds
{
  "name": "minio-java",
  "function": "composeObject(String bucketName, String objectName,List<ComposeSource> composeSources, Map <String,String > headerMap, ServerSideEncryption sseTarget)",
  "args": "size: 6 MB & 6 MB",
  "duration": 11421,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code=NotImplemented, message=A header you provided implies functionality that is not implemented, bucketName=minio-java-test-qk8dk2, objectName=minio-java-test-n7c6q0, resource=/minio-java-test-qk8dk2/minio-java-test-n7c6q0, requestId=1601835C4C823F0E, hostId=e82516ce-4150-4dd9-bf39-455c67ec2ea9)\nrequest={method=PUT, url=http://minio-c3.minio.io:30686/minio-java-test-qk8dk2/minio-java-test-n7c6q0?uploadId=b494ae7d175eac5d&partNumber=1, headers=x-amz-copy-source-if-match: b64bc1a63ef426ffaa50a99de6d54cbf-2\nx-amz-copy-source: minio-java-test-qk8dk2/minio-java-test-r7kchv\nAccept-Encoding: identity\nHost: minio-c3.minio.io:30686\nUser-Agent: MinIO (amd64; amd64) minio-java/dev\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20200331T221616Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20200331/us-east-1/s3/aws4_request, SignedHeaders=accept-encoding;host;x-amz-content-sha256;x-amz-copy-source;x-amz-copy-source-if-match;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=501, headers=Accept-Ranges: bytes\nContent-Length: 406\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nServer: MinIO/DEVELOPMENT.GOGET\nVary: Origin\nX-Amz-Request-Id: 1601835C4C823F0E\nX-Xss-Protection: 1; mode=block\nDate: Tue, 31 Mar 2020 22:16:16 GMT\n}\n >>> [io.minio.MinioClient.executeReq(MinioClient.java:1220), io.minio.MinioClient.execute(MinioClient.java:1082), io.minio.MinioClient.executePut(MinioClient.java:1446), io.minio.MinioClient.executePut(MinioClient.java:1468), io.minio.MinioClient.uploadPartCopy(MinioClient.java:2619), io.minio.MinioClient.composeObject(MinioClient.java:2572), FunctionalTest.composeObject_test1(FunctionalTest.java:2587), FunctionalTest.runTests(FunctionalTest.java:3396), FunctionalTest.main(FunctionalTest.java:3514)]"
}

Executed 9 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by harshavardhana** [comment]
PTAL @kanagarajkm @kaankabalak @rushenn

**Comment by harshavardhana** [comment]
please fix the build issues here @tweigel-dev

**Comment by tweigel-dev** [comment]
hmm, 
I think the pipeline break have no connection to my code changes, because other PR get the same error.
`https://travis-ci.org/github/minio/minio/builds/670012877?utm_medium=notification&utm_source=github_status `
My changes are only at the browser code and not at the golang part. 
But the fail happens at the cmd part
`github.com/minio/minio/cmd/gateway/`

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#3b1bb7cc-dd47-4b47-a3b9-435c0a76beab) |
| mint-gateway-azure.sh | :x: [more...](#7aac34e3-73f2-4d19-a2d2-ebcc6e6f6fc4) |
<a name="7aac34e3-73f2-4d19-a2d2-ebcc6e6f6fc4"></a>
### 9239-8f33301/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev4.minio.io:30101
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 9297b0d2bd10:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 17 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 9 minutes and 24 seconds
(4/15) Running aws-sdk-ruby tests ... done in 20 seconds
(5/15) Running awscli tests ... FAILED in 33 seconds
{
  "name": "awscli",
  "duration": 2166,
  "function": "aws --endpoint-url http://minio-dev4.minio.io:30101 s3api put-object --body /mint/data/datafile-1-kB --bucket \nAn error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it. --key datafile-1-kB\n",
  "status": "FAIL",
  "error": "usage: aws [options] <command> <subcommand> [<subcommand> ...] [parameters]To see help text, you can run:  aws help  aws <command> help  aws <command> <subcommand> helpUnknown options: occurred, (BucketAlreadyOwnedByYou), when, calling, the, CreateBucket, operation:, Your, previous, request, to, create, the, named, bucket, succeeded, and, you, already, own, it., error"
}

Executed 4 out of 15 tests successfully.
```
<a name="3b1bb7cc-dd47-4b47-a3b9-435c0a76beab"></a>
### 9239-8f33301/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c4.minio.io:30939
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 6311ac011e23:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... FAILED in 5 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-sv6u06zx9omcxg",
    "objectName": "mkcida6x0bxteujc6kp9uf0rjiqk0l"
  },
  "duration": 747,
  "error": "RequestError: send request failed\ncaused by: Put http://minio-c4.minio.io:30939/aws-sdk-go-test-sv6u06zx9omcxg/mkcida6x0bxteujc6kp9uf0rjiqk0l?tagging=: EOF",
  "function": "testObjectTagging",
  "message": "AWS SDK Go PUTObjectTagging expected to success but got RequestError",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by harshavardhana** [comment]
> PTAL @kanagarajkm @kaankabalak @rushenn

This is a nice work can you guys take a look?

**Comment by kaankabalak** [comment]
> > PTAL @kanagarajkm @kaankabalak @rushenn
> 
> This is a nice work can you guys take a look?

Thanks for the PR @tweigel-dev, I will take a look at it as soon as possible.

File: browser/app/js/objects/PreviewObjectModal.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                  src={this.state.url}
+                  style={{ display: "block", width: "100%" }}
+                  />)
+                
+                }
+              
+            </div>
+          </ModalBody>
+          <div className="modal-footer">
+            {<button className="btn btn-link" onClick={hidePreviewModal}>
```
Comment (kanagarajkm) [inline browser/app/js/objects/PreviewObjectModal.js]: curly braces `{` are not required before and after `<button`

File: browser/app/js/objects/PreviewObjectModal.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+/*
+ * MinIO Cloud Storage (C) 2018 MinIO, Inc.
```
Comment (kanagarajkm) [inline browser/app/js/objects/PreviewObjectModal.js]: Could you change the year to 2020

**Review by kanagarajkm — COMMENTED** [review]
@tweigel-dev thanks for the contribution. Sorry for the delay in review.

Looks good to me, few minor comments.

Could you please format the changed files using prettier and push.

**Comment by kaankabalak** [comment]
Thanks once again for the PR @tweigel-dev, I can also approve once you have formatted the files 👍

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-fs.sh | :x: [more...](#abe65c30-4c20-41e6-8f09-74ffe14703d9) |
| mint-gateway-azure.sh | :x: [more...](#d69ad808-3816-4963-b5ff-780881794eef) |
<a name="abe65c30-4c20-41e6-8f09-74ffe14703d9"></a>
### 9239-e4705fe/mint-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev7.minio.io:31295
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp fbe81065a13c:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 1 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 41 seconds
(4/15) Running aws-sdk-ruby tests ... done in 2 seconds
(5/15) Running awscli tests ... done in 1 minutes and 6 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 25 seconds
(8/15) Running minio-dotnet tests ... done in 28 seconds
(9/15) Running minio-go tests ... done in 11 minutes and 43 seconds
(10/15) Running minio-java tests ... FAILED in 20 seconds
{
  "name": "minio-java",
  "function": "composeObject(String bucketName, String objectName,List<ComposeSource> composeSources, Map <String,String > headerMap, ServerSideEncryption sseTarget)",
  "args": "with offset and length.",
  "duration": 143,
  "status": "FAIL",
  "error": "java.io.EOFException >>> [okio.RealBufferedSink.write(RealBufferedSink.java:115), io.minio.HttpRequestBody.writeTo(HttpRequestBody.java:76), okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.java:83), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:45), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121), okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:94), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121), okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:125), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121), okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:264), okhttp3.RealCall.execute(RealCall.java:93), io.minio.MinioClient.execute(MinioClient.java:1078), io.minio.MinioClient.execute(MinioClient.java:1217), io.minio.MinioClient.executePut(MinioClient.java:1391), io.minio.MinioClient.executePut(MinioClient.java:1405), io.minio.MinioClient.putObject(MinioClient.java:3793), io.minio.MinioClient.putObject(MinioClient.java:3849), io.minio.MinioClient.putObject(MinioClient.java:3936), FunctionalTest.composeObject_test2(FunctionalTest.java:2952), FunctionalTest.runTests(FunctionalTest.java:3852), FunctionalTest.main(FunctionalTest.java:3963)]"
}

Executed 9 out of 15 tests successfully.
```
<a name="d69ad808-3816-4963-b5ff-780881794eef"></a>
### 9239-e4705fe/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev6.minio.io:32291
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp b8d47740cf36:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 8 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 9 minutes and 3 seconds
(4/15) Running aws-sdk-ruby tests ... done in 19 seconds
(5/15) Running awscli tests ... done in 3 minutes and 6 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 6 minutes and 36 seconds
(8/15) Running minio-dotnet tests ... done in 2 minutes and 0 seconds
(9/15) Running minio-go tests ... done in 20 minutes and 16 seconds
(10/15) Running minio-java tests ... done in 31 minutes and 35 seconds
(11/15) Running minio-js tests ... FAILED in 3 minutes and 51 seconds
{
  "name": "minio-js",
  "function": "listObjectsV2WithMetadata(bucketName, prefix, recursive, startAfter)",
  "args": "bucketName:minio-js-test-6a6637e0-07ab-4a44-92b2-57dd53a32335, recursive:true",
  "duration": 213,
  "status": "FAIL",
  "error": "TypeError: Cannot read property '0' of undefined at /mint/run/core/minio-js/node_modules/minio/dist/main/xml-parsers.js:414:42 at Array.forEach (<anonymous>) at parseListObjectsV2WithMetadata (node_modules/minio/dist/main/xml-parsers.js:409:21) at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:88:19) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (_stream_readable.js:673:10) at endReadableNT (_stream_readable.js:1201:12) at processTicksAndRejections (internal/process/task_queues.js:84:21)"
}

Executed 10 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally
Error: No such image: minio/minio:9239-e4705fe

**Comment by tweigel-dev** [comment]
simple CI seems broken ? Cant see any output.

**Comment by harshavardhana** [comment]
triggered a rebuild @tweigel-dev also always rebase your changes with master

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#dff03552-a906-4eb7-b3a3-8630af27146f) |
<a name="dff03552-a906-4eb7-b3a3-8630af27146f"></a>
### 9239-f7bfb3f/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev4.minio.io:31036
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 36eb66925fad:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 8 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 9 minutes and 11 seconds
(4/15) Running aws-sdk-ruby tests ... done in 19 seconds
(5/15) Running awscli tests ... FAILED in 10 seconds
{
  "name": "awscli",
  "duration": 1488,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it."
}

Executed 4 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by tweigel-dev** [comment]
Hay,
thanks for your feedback. it should be implemented and the pipeline is passed. is there something else I have to do? @harshavardhana @kaankabalak  @kanagarajkm @rushenn

**Comment by harshavardhana** [comment]
@rushenn @kaankabalak @kanagarajkm anything pending here?

**Comment by kaankabalak** [comment]
@tweigel-dev, could you not include the semicolons at the end while running prettier? Thanks once again!

**Comment by tweigel-dev** [comment]
I removed the ; at the files touched by the changes with an online formatter. 

the format script at the package.json never passed the hole code and would change nearly any file. maybe it can be useful to configure eslint and integrate it at the pipeline. with the .eslintrc it is possible to configure such rules. 
also it is possible to autofix lightweight lint errors like semicolons and indents.

Thanks for your feedback. 
I hope all minor comments are resolved now.

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#eb2bcd39-8d82-44fb-adc5-5258d76d915a) |
<a name="eb2bcd39-8d82-44fb-adc5-5258d76d915a"></a>
### 9239-296712d/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev4.minio.io:31688
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 086d7089d495:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 7 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 9 minutes and 13 seconds
(4/15) Running aws-sdk-ruby tests ... done in 18 seconds
(5/15) Running awscli tests ... FAILED in 3 seconds
{
  "name": "awscli",
  "duration": 1400,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it."
}

Executed 4 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#0737d67a-c2a7-4e4a-ba34-b4cdc0fcf4c0) |
<a name="0737d67a-c2a7-4e4a-ba34-b4cdc0fcf4c0"></a>
### 9239-bcbac1f/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c3.minio.io:31802
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 2c91aaefd12a:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 7 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 9 minutes and 18 seconds
(4/15) Running aws-sdk-ruby tests ... done in 17 seconds
(5/15) Running awscli tests ... FAILED in 1 minutes and 38 seconds
{
  "name": "awscli",
  "duration": 1478,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it."
}

Executed 4 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by kanagarajkm — APPROVED** [review]

**Review by kaankabalak — APPROVED** [review]

**Review by rushenn — APPROVED** [review]

**Comment by ihostage** [comment]
Related with https://github.com/minio/minio/issues/3835

**Pull request merged**